src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import VirtualizedGrid from "../VirtualizedGrid";
+import Button from "../../Button/Button";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { generateItems, itemRenderer } from "./virtualizedGrid.components";
+
+<Meta
+  title="Navigation/VirtualizedGrid"
+  component={ VirtualizedGrid }
+/>
+
+<!--- Component template -->
+
+export const virtualizedGridTemplate = createComponentTemplate(VirtualizedGrid);
+
+<!--- Component documentation -->
+
+# VirtualizedGrid
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Feedback](#feedback)
+
+## Overview
+VirtualizedGrid is a component which only renders visible grid items, it is a logic component and doesn't change and look and feel
+
+Under the hood we are using - [react-window](https://github.com/bvaughn/react-window) and [react-virtualized-auto-sizer](https://github.com/bvaughn/react-virtualized-auto-sizer)
+<Canvas>
+    <Story name="Overview" args={{}} >
+        <div
+          style={{
+            width: 430,
+            height: 300,
+            overflow: "hidden",
+            display: "flex",
+            alignItems: "center"
+          }}
+        >
+          <div style={{ width: 600, height: "100%" }}>
+            <VirtualizedGrid
+              items={generateItems(50, 100, 1000)}
+              itemRenderer={itemRenderer}
+              getColumnWidth={()=> {return 100}}
+              getRowHeight={()=> {return 50}}
+              id="Knobs"
+              scrollDuration={200}
+            />
+          </div>
+        </div>
+    </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ VirtualizedGrid } />
+
+## Usage
+<UsageGuidelines guidelines={[
```

Comment:
my suggestion is to replace this one usage rule with the following two rules:
- Use this component to implement a grid with many items or when each item render has heavy calculations.
- Rendering only the visible grid items instead of all the grid's items will create better performance and a smoother experience for users while using the grid.